### ISSUE TITLE
Deprication warning with RSpes 3.4.2

### ISSUE THREAD

**Issue by fnordfish** [udzura/rspec-padrino#13]
When using with RSpec 3.4.2 I'm getting those deprecation warnings:

```
`failure_message_for_should_not` is deprecated. Use `failure_message_when_negated` instead. Called from /home/vagrant/.gem/ruby/2.3.0/gems/rspec-padrino-0.2.2/lib/rspec/padrino/matchers/redirect_to.rb:26:in `block in <module:RedirectTo>'.

`failure_message_for_should` is deprecated. Use `failure_message` instead. Called from /home/vagrant/.gem/ruby/2.3.0/gems/rspec-padrino-0.2.2/lib/rspec/padrino/matchers/redirect_to.rb:22:in `block in <module:RedirectTo>'.
```

After having a fast look into the RSpec changelog, this was added with 3.0.0.beta2.

**Comment by namusyaka** [comment]
closed via #15

**Comment by namusyaka** [comment]
Thanks for your report.
